alphastats/plots/ClusterMap.py

```diff
@@ -62,7 +63,7 @@ def _prepare_df(self):
             significant_proteins = anova_df[anova_df["ANOVA_pvalue"] < 0.05][
                 self.index_column
             ].to_list()
-            df = df[significant_proteins]
+            df = df[significant_proteins]  # TODO bug?
```

Comment:
No, this is intended, however it would be better if the statistical results could be selected here, instead of running anova (intransparently).

Reply:
I was referring to the fact that `df` is not used after this line ..